### ISSUE TITLE
FMV 4:3 Auto-Switch even on little videos

### ISSUE THREAD

**Issue by Catarax** [PCSX2/pcsx2#1428]
FMV 4:3 auto-switch works very well, however it applies also in-game when there is a little video playing and it makes the game even more embarrassing.

It should be only done one when a FMV play in fullscreen.
So the idea is to do a 4:3 switch only if the size of the video is big enough!

**Select Screen (Stretched 16:9)**
![screenshot_2](https://cloud.githubusercontent.com/assets/3964507/16225504/2ded5546-37a7-11e6-80b0-a8bc53ccb857.png)

**Stage Select (4:3 due to little FMV playing in the circle and should not here)**
![screenshot_1](https://cloud.githubusercontent.com/assets/3964507/16225507/2f222b3a-37a7-11e6-885c-a70fc86971a8.png)

On game like Guitar Hero 5, the game-play is forced to 4:3 too because the public is a FMV.

**Comment by FlatOutPS2** [comment]
Then the same should apply for the switch to software game fix. Can you confirm that?

**Comment by Catarax** [comment]
What do you mean by the switch for software game fix? Let me know where I can enable it, thanks.

**Comment by ssakash** [comment]
@FlatOutPS2 

Try to initialize the switch only when the drawcall rate is too low (around 0). FMV's just upload the image in rendering target without any rendering so technically draw call rate should be null. This way you prevent enabling the switch on in-game scenes.

**Comment by ssakash** [comment]
> What do you mean by the switch for software game fix? Let me know where I can enable it, thanks.

It's located at GameFixes tab of Emulation settings dialog. though it's highly likely that it also shares the same bug.

**Comment by ramapcsx2** [comment]
Looking at draw calls will work in many games but some (like FFX-2) would probably go undetected then.

**Comment by Catarax** [comment]
Yes, the same happen when the video is read with the software game fix.

**Comment by gregory38** [comment]
Final Fantasy video are points rendered so they uses some draw calls. However I'm not sure they uses lots of draw calls. (maybe skip point draw call)

**Comment by Mrlinkwii** [comment]
Can this be checked on latest nightly?

**Comment by DigitalMajestic** [comment]
Still an issue, at least with ATV Off Road Fury 4 (main menu has an FMV going in the background).
